src/components/CellLineTable/DiseaseTableColumns.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { Flex } from "antd";
+import classNames from "classnames";
+
+import {
+    Clone,
+    UnpackedDiseaseCellLine,
+    UnpackedGene,
+    UnpackedNormalCellLine,
+} from "../../component-queries/types";
+import { formatCellLineId, getCloneSummary } from "../../utils";
+import GeneDisplay from "../GeneDisplay";
+import ParentalLineModal from "../ParentalLineModal";
+import CloneSummary from "../CloneSummary";
+
+import {
+    cellLineIdColumn,
+    certificateOfAnalysisColumn,
+    obtainLineColumn,
+} from "./SharedColumns";
+import { smBreakPoint, mdBreakpoint, CellLineColumns } from "./types";
+
+const {
+    clones,
+    lastColumn,
+    snpColumn,
+} = require("../../style/table.module.css");
+
+export const getDiseaseTableColumns = (
```

Comment:
these are pretty much unchanged from when they were previously in `DiseaseTable`. I factored out the columns so i can have a unified `CellLineTable` that can take either type of column